environments/python/builder/Dockerfile

```diff
@@ -2,7 +2,7 @@ ARG BUILDER_IMAGE=fission/builder:latest
 FROM ${BUILDER_IMAGE}
 
 RUN apk update
-RUN apk add --no-cache python3 python3-dev build-base
+RUN apk add --no-cache python3 python3-dev gcc build-base bash py-pip
```

Comment:
Sorry, why this change is required? Ideally, the environment image should not have to change at all